### PR TITLE
Allow specifying custom messages on specific networks

### PR DESCRIPTION
**Pull request by kewisch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,21 @@ Using:
 </ul>
 ```
 
+You can also define a custom url, title, description or image specific to the the service:
```
Comment (rpignolet) [inline README.md]: `the` is duplicate

**Comment by rpignolet** [comment]
Good for me :+1:

**Comment by chamerling** [comment]
:+1:

**Comment by GrahamLinagora** [comment]
Seems good to me

**Pull request merged**